database/database.go

```diff
@@ -0,0 +1,65 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nimblehq/mark-ic/helpers"
+	"github.com/nimblehq/mark-ic/helpers/log"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pressly/goose/v3"
+	"github.com/spf13/viper"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var database *gorm.DB
+
+func InitDatabase(databaseURL string) {
+	db, err := gorm.Open(postgres.Open(databaseURL), &gorm.Config{})
+	if err != nil {
+		log.Fatalf("Failed to connect to %v database: %v", gin.Mode(), err)
+	} else {
+		viper.Set("database", db)
+		log.Println(strings.Title(gin.Mode()) + " database connected successfully.")
+	}
```

Comment:
Interesting suggestion from GitHub, we might want to open a quick and easy fix in the Gin-template to replace this `Title` call 😇